Source/DataSources/KmlTour.js

```diff
@@ -1,12 +1,20 @@
 import defined from "../Core/defined.js";
 import Event from "../Core/Event.js";
 /**
+ * Describes a KmlTour, which uses KmlTourFlyTo, KmlTourWait, and KmlTourSoundCues to
+ * guide the camera to a specified destinations on given time intervals.
+ *
  * @alias KmlTour
  * @constructor
  *
  * @param {String} name name parsed from KML
  * @param {String} id id parsed from KML
- * @param {Array} playlist array with KMLTourFlyTos, KMLTourWaits and KMLTourSoundCues
+ * @param {Array} playlist array with KmlTourFlyTos, KmlTourWaits and KmlTourSoundCues
+ *
+ * @see KmlTourFlyTo
+ * @see KmlTourWait
+ *
+ * @demo {@link https://sandcastle.cesium.com/?src=KML%20Tours.html KML Tours}
```

Comment:
The generated doc looked ok but generally there's a `|` symbol between the link and name

```suggestion
 * @demo {@link https://sandcastle.cesium.com/?src=KML%20Tours.html|KML Tours}
```